client/src/controllers/SessionController.ts

```diff
@@ -197,12 +195,32 @@ export class SessionController extends Controller<HTMLElement> {
   }
 
   wDialogOutletConnected(): void {
+    // Attach the event listener to the buttons that will be intercepted.
+    // Do it here instead of in connect() so hopefully this includes any buttons
+    // that are inside a dialog that is connected after this controller
+    // (e.g. the schedule publishing dialog).
+    this.interceptTargets.forEach((button) => {
+      // Match the event listener configuration of workflow-action that uses
+      // capture so we can intercept the workflow-action's listener.
+      button.addEventListener('click', this.showConfirmationDialog, {
+        capture: true,
+      });
+    });
+
     this.wDialogOutlet.element.addEventListener(
       'w-dialog:confirmed',
       this.confirmAction,
     );
   }
 
+  wDialogOutletDisconnected(): void {
+    this.interceptTargets?.forEach((button) => {
+      button.removeEventListener('click', this.showConfirmationDialog, {
+        capture: true,
+      });
+    });
+  }
```

Comment:
Moving this from `disconnect()` to here for consistency with how we moved the `addEventListener` to `wDialogOutletConnected`